sql/moz-fx-data-shared-prod/stripe/itemized_payout_reconciliation/view.sql

```diff
@@ -45,17 +48,41 @@ charge_states AS (
   WHERE
     cards.country IN ("US", "CA")
 ),
+subscriptions AS (
+  SELECT
+    subscription_id,
+    plan_id,
+    product_id,
+  FROM
+    `moz-fx-data-shared-prod`.subscription_platform.stripe_subscriptions
+),
 enriched AS (
   SELECT
-    report.*,
+    report.* REPLACE(
+      customer.shipping_address_country as shipping_address_country,
+      customer.shipping_address_state as shipping_address_state,
+      customer.shipping_address_postal_code as shipping_address_postal_code
```

Comment:
i added this over the existing columns because the existing columns are always empty, but I also left the old country calculations in place because the new values are only present for ~1/8th of customers.